app/components/raw-config.js

```diff
@@ -0,0 +1,40 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { isInternal, presentedPath, fileNameWithoutSha } from 'travis/utils/format-config';
+import { later } from '@ember/runloop';
+
+export default Component.extend({
+  copied: false,
+  baseYmlName: '.travis.yml',
+  externalLinks: service(),
+
+  buttonLabel: computed('copied', function () {
+    let source = this.get('rawConfig.source');
```

Comment:
This computed property seems to depend on `rawConfig.source` too. So it should be listed in dependent keys so that the property re-compute properly:
```js
buttonLabel: computed('copied', 'rawConfig.source', function () {
})
```